blog/2024-02-14-flet-adaptive-and-custom-controls.md

```diff
@@ -28,7 +28,7 @@ sudo apt install libmpv-dev mpv
 
 Adaptive controls allow writing apps with a single code base which look and behave differently depending on the platform they are running on.
 
-To the date Flet provides [11 adaptive controls](/roadmap#adaptive-and-cupertino-controls). To make control adaptive you should set its `adaptive` property to `True`.
+To the date Flet provides 11 adaptive controls. To make control adaptive you should set its `adaptive` property to `True`.
```

Comment:
Removed a link.

Reply:
That link was breaking the build because It doesn't exist anymore.
Ok, I will modify to [this](https://flet.dev/docs/getting-started/adaptive-apps#material-and-cupertino-controls).